data/json/vehicles/vehicles.json

```diff
@@ -5,14 +5,19 @@
     "name": "custom",
     "//": "Don't remove this vehicle definition!",
     "blueprint": [ "H" ],
-    "parts": [ { "x": 0, "y": 0, "part": "frame_vertical_2" } ]
+    "parts": [ {
+        "x": 0,
+        "y": 0,
+        "part": "frame_vertical_2"
+      }
```

Comment:
Another huge red flag. This file only has the dummy vehicles.
`data/json/vehicles/cars.json` isn't very readable at the moment, but after this formatting, it would look worse than an unformatted freeform mess.